src/main/webapp/app/exercises/shared/presentation-score/presentation-score.component.ts

```diff
@@ -29,6 +29,14 @@ import { Authority } from 'app/shared/constants/authority.constants';
             </div>
         </ng-container>
     `,
+    styles: [
+        `
+            .form-check-label {
+                font-size: 15px;
+                font-weight: bolder;
+            }
+        `,
+    ],
```

Comment:
Could we use a `styleUrls` here instead? Seems like an unnecessary duplication here to me.

Reply:
I agree doing it in two places will create the problem again in the future.